src/OptiHPLCHandler/empower_api_core.py

```diff
@@ -10,6 +10,11 @@
 logger = logging.getLogger(__name__)
 
 
+class EmpowerResponse(NamedTuple):
+    result: Optional[dict]
+    message: Optional[str]
```

Comment:
Do we want to allow these to be optional? We could just retrun an empty dict and an empty string if there isn't anything there, that would be more interoperable.

Is there ever an important difference between the API not returning content, and it returning empty content?